### DISCUSSION TITLE
Can we attach maxDistance, maxDuration, maxStoresToVisit for particular van/route?

### DISCUSSION THREAD

**Discussion by SruthyJohn** [google/or-tools#3140] [Routing (and legacy CP) questions]
Hi Team,
Is there a way to add maxDistance or maxDuration or maxStoresToVisit for each vehicle as we attach Demands to each vehicle?

I need routes in such a way that it should adhere to any of the maxDistance, maxDuration, maxStores, capacity/demand constraint of the route/van.

**Comment by Mizux** [comment]
create a RoutingDimension per dimension and set the vehicle max capacity accordingly ?

i.e. You can have any number of dimensions you want/need
